frontend/components/DocumentFilters/DocumentFilters.jsx

```diff
@@ -140,9 +166,8 @@ export default class DocumentFilters extends Component {
     const {updateUrlParams} = this.props
 
     let validFilters = filters.filter(entry => {
-      let isValid = Object.keys(entry).filter(key => {
-        return Object.is(entry[key], null)
-      })
+      let isValid = Object.keys(entry)
+        .filter(key => Object.is(entry[key], null))
```

Comment:
I see we're using `Object.is` [here](https://github.com/dadi/publish/blob/master/frontend/middleware/router.js#L57) as well, which will cause trouble in IE. We should probably revisit that.